Code/Geometry/Transform3D.h

```diff
@@ -56,15 +56,18 @@ class RDKIT_RDGEOMETRYLIB_EXPORT Transform3D
 
   /*! \brief set the rotation matrix
    *
-   * The rotation matrix is set to rotation by th specified angle
+   * The rotation matrix is set to rotation by the specified angle
    * about the specified axis
    */
   void SetRotation(double angle, AxisType axis);
 
   /*! \brief set the rotation matrix
    *
-   * The rotation matrix is set to rotation by th specified angle
-   * about an arbitrary axis
+   * The rotation matrix is set to rotation by the specified angle
+   * about an arbitrary axis.
+   * NB The axis must be a unit vector, otherwise scaling will also occur.
```

Comment:
I'd prefer to phrase this:
```suggestion
   * Note: if the axis is not a unit vector scaling will also occur.
```